MANIFEST.in

```diff
@@ -1,2 +1,6 @@
 include versioneer.py
 recursive-include uarray *.py
+include LICENSE
+recursive-include uarray *.cxx
```

Comment:
this may leave something out. why not `recursive-include uarray *`? and then make sure to `prune` or `global-exclude` tempfiles

Reply:
hmm at the moment there's only one file though